tests/integration/integration_test.go

```diff
@@ -940,6 +967,136 @@ status: {}
 
 			Expect(podList.Items[0].Spec.Containers[0].Image).To(Equal("docker.io/bitnami/nginx:1.19.10-debian-10-r35"))
 		})
+		When("there is a depending chart", func() {
```

Comment:
this only tests charts right now. not kustomize... eventually, I'll expand the profile to be more complex